.github/workflows/rspec_rubocop.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
```

Comment:
generally i'd expect actions to run on every branch, and certainly on every PR

(separately, you may want to rename the master branch to main)

Reply:
Yep! Getting there, just iterating for now.